EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76450
+dood.so$$script[tag-content="adblock"][min-length="500"][max-length="1000"]
+!+ PLATFORM(ios, ext_android_cb, ext_safari, ext_ff)
+@@||dood.so^$generichide
```

Comment:
`dood.*#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display: block !important; }` this is working fine on my end. I'll make changes directly.

Reply:
@zloyden It get overwritten on my end.

Reply:
@zloyden It's Okay I don't mind. But can you make that rule a generic one?